index.html

```diff
@@ -9,68 +9,129 @@
-               <button class="button">
+            <div class="menu__button-wrap">
+               <button class="button menu__button">
                   <div class="menu__button-text">Contact us</div>
-                  <img src="img/header/arrow-down.png" alt="">
+
+                  <img class="menu__button-img" src="img/header/arrow-down.png" alt="">
                </button>
             </div>
          </div>
 
          <div class="header__language dropdown">
-            <button class="header__button" type="button"
-               data-bs-toggle="dropdown" aria-expanded="false">
+            <button class="header__button" type="button" data-bs-toggle="dropdown" aria-expanded="false">
                <img class="header__button-img" src="img/header/flag UK.png" alt="EN">
             </button>
 
             <ul class="dropdown-menu">
-               <li><a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag UA.png" alt=""></a></li>
-               <li><a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag FR.png" alt=""></a></li>
-               <li><a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag GR.png" alt=""></a></li>
+               <li>
+                  <a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag UA.png"
+                        alt=""></a>
+               </li>
+
+               <li>
+                  <a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag FR.png"
+                        alt=""></a>
+               </li>
+
+               <li>
+                  <a class="dropdown-item" href="#"><img class="header__button-img" src="img/header/flag GR.png"
+                        alt=""></a>
+               </li>
             </ul>
          </div>
       </div>
    </header>
 
+   <main class="main">
+      <section class="first">
+         <div class="container first__container">
+            <div class="first__get-start flex">
+               <div class="get-start__info">
+                  <h1 class="get-start__title">SOLUZIONI <span class="get-start__span">DIGITALI</span> <span
+                        class="get-start__span">SMART</span> E INNOVATIVE
+                  </h1>
+
+                  <p class="get-start__text">Competenza, innovazione e creatività: ideiamo nuove esperienze digitali
+                     personalizzate per le esigenze della tua azienda.
+                  </p>
+
+                  <button class="button get-start__button">
+                     <div class="get-start__button-text">Get Started</div>
+                     <img class="get-start__button-img" src="img/header/arrow-down.png" alt="">
+                  </button>
+               </div>
+
+               <img class="img_get-start" src="img/main/logo big.svg" alt="big logo">
+            </div>
+
+            <div class="first__about-us flex">
```

Comment:
If you want to add a custom class to the element and use also bootstrap/own created classes, add a custom class as the last because there can be cases when you will not be able to rewrite the class in the future. Let me know if it's not clear and I will explain on the call.